spokestack/tts/tts_manager.py

```diff
@@ -0,0 +1,35 @@
+"""
```

Comment:
it would be simpler just to call this module `spokestack.tts.manager` instead of `spokestack.tts.tts_manager`